pages/common/atq.md

```diff
@@ -0,0 +1,15 @@
+# atq
+
+> Show jobs scheduled by at or batch commands.
+
+- Show the current user's scheduled jobs:
+
+`atq`
+
+- Show jobs from queue named 'a' (queues have one character name):
+
+`atq -q {{a}}`
+
+- Show jobs of all users (run as su):
```

Comment:
Would "run as superuser" be correct? If so I'd suggest that, as it's more intuitive/explicit.